its/projects/VueWithAspBackend/NuGet.Config

```diff
@@ -2,8 +2,14 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="Repox Artifactory (proxy)" value="https://repox.jfrog.io/artifactory/api/nuget/nuget" />
-= </packageSources>
+    <add key="Repox" value="https://repox.jfrog.io/artifactory/api/nuget/v3/nuget/index.json" protocolVersion="3" />
```

Comment:
What about deleting this file and adding `domaindrivendev` to the root, with a small comment explaining that it's used in this project... just like every other owner has in the root?

Reply:
The file is copied and used by the integration test to a temp directory. I do not want to do a rework at this moment. The goal is to fix the problem we have to make the CI green.